client/js/app.modules/main/ng-header-block.directive.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @ngInject
+ * */
+ module.exports = function(){
+     return {
+         restrict: 'E',
+         transclude: true,
+         scope: {
+            isProjectView: '@',
```

Comment:
isProjectView used only for adding classes. I think you should delete it and do the same by adding class when you use this directive. (+header shouldn't be aware of such kind of data)